doc/tutorial/fraud-analysis.md

```diff
@@ -0,0 +1,51 @@
+# Customize Data Analysis on SQLFlow Tutorial
+
+Data analysis can help us understand what is in the dataset and the
+characteristics of the data.
+
+Data binning is a commonly-used data analysis technique. It groups continous values
+into a small number of discretized bins. We will get the distribution of the
+data from the binning result.
+
+We can use SQLFlow TO RUN statement to call the SQLFlow runnable which is
+released in the form of Docker image. SQLFlow provides some premade runnables
+in sqlflow/runnable including the binning runnable. Please use the following
+SQL statement to do the data binning. All the table columns specified in the
```

Comment:
Briefly explain how the binning was done here?

Reply:
Good idea, will add it in the next PR.